tests/Behat/RdfEntityContext.php

```diff
@@ -0,0 +1,205 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_content\Behat;
+
+use Behat\Behat\Hook\Scope\AfterScenarioScope;
+use Behat\Behat\Hook\Scope\BeforeScenarioScope;
+use Drupal\DrupalExtension\Context\RawDrupalContext;
+
+/**
+ * Defines step definitions specifically for testing the RDF entities.
+ *
+ * This is needed due to issues with the Kernel testing of the RDF entity module
+ * in Drupal 8.6.
+ */
+class RdfEntityContext extends RawDrupalContext {
+
+  /**
+   * An array of RDF entities we need to keep track to delete.
+   *
+   * @var \Drupal\rdf_entity\RdfInterface[]
+   */
+  protected $rdfEntities = [];
+
+  /**
+   * The Configuration context.
+   *
+   * @var \Drupal\DrupalExtension\Context\ConfigContext
+   */
+  protected $configContext;
+
+  /**
+   * Before scenario hook get the ConfigContext.
+   *
+   * @param \Behat\Behat\Hook\Scope\BeforeScenarioScope $scope
+   *   The Hook scope.
+   *
+   * @BeforeScenario
+   */
+  public function gatherConfigContext(BeforeScenarioScope $scope): void {
+    $environment = $scope->getEnvironment();
+    $this->configContext = $environment->getContext('Drupal\DrupalExtension\Context\ConfigContext');
+  }
+
+  /**
+   * After scenario hook to delete the RDF entities.
+   *
+   * @param \Behat\Behat\Hook\Scope\AfterScenarioScope $scope
+   *   The Hook scope.
+   *
+   * @AfterScenario @rdf-test
+   */
+  public function afterScenarioRdfCleanUp(AfterScenarioScope $scope): void {
+    if (empty($this->rdfEntities)) {
+      return;
+    }
+
+    foreach ($this->rdfEntities as $rdf) {
+      $rdf->delete();
+    }
+  }
+
+  /**
+   * Before scenario hook sets a default Provenance URI on the site.
+   *
+   * @param \Behat\Behat\Hook\Scope\BeforeScenarioScope $scope
+   *   The Hook scope.
+   *
+   * @BeforeScenario @rdf-provenance
+   */
+  public function setDefaultProvenance(BeforeScenarioScope $scope): void {
+    $this->configContext->setConfig('oe_content.settings', 'provenance_uri', 'http://example.com');
+  }
+
+  /**
+   * Configures the Provenance URI.
+   *
+   * @Given the site Provenance URI is set to :uri
+   */
+  public function theSiteProvenanceUriIsSetTo($uri): void {
+    $this->configContext->setConfig('oe_content.settings', 'provenance_uri', $uri);
+  }
+
+  /**
+   * User creation and login for creating RDF entities.
+   *
+   * A custom step that creates a user that has the permissions to create and
+   * view RDF entities of a give type.
+   *
+   * @Given I am logged in with a user that can create and view :bundle RDF entities
+   */
+  public function iAmLoggedInWithUserThatCanCreateAndViewRdfEntityTypes($bundle) {
```

Comment:
Let's follow the naming on the DrupalContext (even if it doesn't contain asserts...) 
```public function assertLoggedInWithRdfEntityTypePermissions($bundle): void {```